keyboards/pulsar/xboard_qs/config.h

```diff
@@ -0,0 +1,110 @@
+/* Copyright 2024 Aplusx Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define WAIT_US_TIMER           GPTD3       //  Encoder 및 RGB ANI가 부드러워짐
+
+// Dynamic EEPROM
```

Comment:
Many of the config options below should be migrated to info.json

Reply:
Thank you.
I'll check and fix it as much as I can

Reply:
I don't know "git" well, and I don't know "QMK" well, so thank you for reviewing it
I checked as much as I could and completed the correction
I'm sure you're busy, but please review it again when the conditions are available